packages/block/src/header.ts

```diff
@@ -325,30 +383,38 @@ export class BlockHeader {
    * Returns the hash of the block header.
    */
   hash(): Buffer {
-    return rlphash(this.raw)
+    const values: Buffer[] = this.raw()
+    return rlphash(values)
   }
 
   /**
-   * Checks if the block header is a genesis header.
+   * Returns a Buffer Array of the raw Buffers in this header, in order
    */
-  isGenesis(): boolean {
-    return this.number.length === 0
+  raw(): Buffer[] {
+    return [
+      this.parentHash,
+      this.uncleHash,
+      this.coinbase,
+      this.stateRoot,
+      this.transactionsTrie,
+      this.receiptTrie,
+      this.bloom,
+      this.difficulty,
+      this.number,
```

Comment:
Now that some of these are BNs, they need to be converted unpadded buffers here.

I am adding a fn `bnToRlp(value: BN)` in https://github.com/ethereumjs/ethereumjs-util/pull/275 but meanwhile you can just do what it does underneath which is `unpadBuffer(value.toArrayLike(Buffer))`